docs/mimxrt/quickref.rst

```diff
@@ -537,6 +537,62 @@ port and LAN(1) for the 1G port.
 
 For details of the network interface refer to the class :ref:`network.LAN <network.LAN>`.
 
+class Encoder -- Quadrature Encoder for i.MXRT MCUs
+---------------------------------------------------
+
+This class provides the Quadrature Encoder Service.
+
+Example usage::
+
+    # Samples for Teensy
+    #
+
+    from machine import Pin, Encoder
+
+    qe = Encoder(0, Pin(0), Pin(1)) # create Quadrature Encoder object
+    qe.value()                      # get current counter values
+    qe.value(0)                     # Set value and cycles to 0
+    qe.init(cpc=128)                # Specify 128 counts/cycle
```

Comment:
Counts per cycle(cpc) is a vague term for me. Should this be the Encoder's Counts Per Revolution(cpr), does it mean the Pulse Per Revolution(ppr)?